src/tooltip/tooltip.js

```diff
@@ -315,6 +317,11 @@ angular.module('ui.bootstrap.tooltip', ['ui.bootstrap.position', 'ui.bootstrap.s
                 $timeout.cancel(transitionTimeout);
                 transitionTimeout = null;
               }
+              
+              if (positionTimeout) {
```

Comment:
This was something I added thinking there was may be a left over timeout and potential memory leak
However, it isn't actually part of the issue, and was more of a memory tweak. 

Although it didn't cause issues in my scenario, it probably requires more testing